query/test/functional/test_endpoints.py

```diff
@@ -4,21 +4,16 @@
 
 import requests
 
+from lib.config import Config
 
 
 class TestQueryService(unittest.TestCase):
 
     def __init__(self, *args, **kwargs):
```

Comment:
seems like you could just do away with this method.

Reply:
👍